### ISSUE TITLE
Endpoints available are filtered by zoom and userAgentFilter

### ISSUE THREAD

**Issue by zachmandeville** [cncf/apisnoop#125]
```feature
Given an array of endpoints for a release
And a sunburst that contains all these endpoints
And a filter for useragents
And a filter for zoom level (the part of the sunburst we've selected to zoom in on)
When a useragent filter is set
Then the array of endpoints is reduced to only those hit by useragents from the useragent filter
And also those covered by the current zoom's selection (if there is any)
And the sunburst displays this reduced endpoint array
```

**Comment by zachmandeville** [comment]
Requires #134

**Comment by zachmandeville** [comment]
implemented in #140
